api/configDB/config.ts

```diff
@@ -1,8 +1,18 @@
 import { Pool } from 'pg';
+import * as winston from 'winston';
 
 const pool = new Pool({
   connectionString: process.env.DATABASE_URI,
   ssl: { rejectUnauthorized: false },
 });
 
+pool

```

Comment:
W całej aplikacji mamy async i await, może przeróbmy to w ten sposób, żeby to ujednolicić?